protocol/contracts/v2/strategies/SkyFarmBaseStrategy.sol

```diff
@@ -0,0 +1,227 @@
+pragma solidity 0.8.20;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Temple (v2/strategies/SkyFarmBaseStrategy.sol)
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IERC4626 } from "@openzeppelin/contracts/interfaces/IERC4626.sol";
+
+import { ITempleBaseStrategy } from "contracts/interfaces/v2/strategies/ITempleBaseStrategy.sol";
+import { AbstractStrategy } from "contracts/v2/strategies/AbstractStrategy.sol";
+import { CommonEventsAndErrors } from "contracts/common/CommonEventsAndErrors.sol";
+
+interface DaiUsds {
```

Comment:
maybe extract this into an external interface file

Reply:
Done